src/server/svr_chk_owner.c

```diff
@@ -390,66 +390,94 @@ authenticate_user(struct batch_request *preq, struct connection *pcred)
  *		this checks the most conditions common to most job batch requests.
  *		It also returns a pointer to the job if found and the tests pass.
  *		If the request is for a single subjob or a range of subjobs (of an
- *		Job Array),  the return job pointer is to the parent Array Job.
+ *		Job Array), alsoe return pointer to the parent Array Job.
  * @par
  *		Depending on what the "jobid" identifies, the following is returned
- *		in the integer pointed to by rc:
+ *		in the integer pointed to by jobtype:
  *	   	IS_ARRAY_NO (0)       - for a regular job
  *	   	IS_ARRAY_ArrayJob (1) - for an Array Job
  *	   	IS_ARRAY_Single (2)   - for a single subjob
  *	   	IS_ARRAY_Range (3)    - for a range of  subjobs
  *
  * @param[in]	jobid	-	Job Id.
  * @param[in,out]	preq	-	job batch request
- * @param[out]	rc	-	Depending on what the "jobid" identifies,
+ * @param[out]	jobtype	-	Depending on what the "jobid" identifies,
  * 						the following is returned
  *						(0)       - for a regular job
  *						(1) - for an Array Job
  *						(2)   - for a single subjob
  *						(3)    - for a range of  subjobs
+ * @param[out]	jobstate	-	the state of the job
+ * @param[out]	ppjob	-	pointer to the job
```

Comment:
This is IN and OUT parameter right ?

Reply:
fixed in rc 11 https://github.com/PBSPro/pbspro/pull/590/commits/1631a47b00ac805440c976a0ee8fb82ec39c2284